app/models/katello/glue/pulp/repo.rb

```diff
@@ -691,6 +705,22 @@ def package_lists_for_publish
          :filenames => filenames.to_set}
       end
 
+      def docker?
```

Comment:
These are moved so all types of repos used by pulp (like content view puppet environment)  can respond to these methods.